spec/features/compression_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+feature 'Compression' do
+  scenario "a visitor has a browser that supports compression", :js do
+    page.driver.allow_url('filepicker.io')
+    page.driver.allow_url('googleapis.com')
+
+    ['deflate','gzip', 'deflate,gzip','gzip,deflate'].each do|compression_method|
```

Comment:
Line is too long. [81/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.